src/styles/blocks/card.scss

```diff
@@ -0,0 +1,62 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
You can set it for all elements (*), so you don't need to reset it manually everywhere